app/models/form_profiles/va21686c.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class FormProfiles::VA21686c < FormProfile
+  def prefill(user)
+    super
+  end
+
+  def metadata
+    {
+      version: 0,
+      prefill: true,
+      returnUrl: '/claimant-information'
```

Comment:
@jbalboni is this right for the `returnUrl`?

Reply:
The first page is `/veteran-information`.